dslink-scala-nest/src/main/scala/com/uralian/nest/service/NestHttpClient.scala

```diff
@@ -74,4 +78,29 @@ class NestHttpClient(config: NestClientConfig) {
   def httpGet[T: Manifest](query: String)
                           (implicit at: AccessToken, as: ResponseAs[T, Nothing], ec: ExecutionContext): Future[T] =
     httpGet(uri"${config.apiUrl}/$query")
+
+  val log = LoggerFactory.getLogger(getClass)
+
+  /**
+    *
+    * @param at
+    * @param ec
+    * @return
+    */
+  def httpStream()(implicit at: AccessToken, ec: ExecutionContext): Future[Source[ByteString, NotUsed]] = {
+
+    val request = sttp.auth.bearer(at.token).header("Accept", "text/event-stream")
```

Comment:
I'm just being picky, but:

- We can use `com.softwaremill.sttp.HeaderNames` trait or object for `"Accept"`
- We can use `com.softwaremill.sttp.MediaTypes` trait or object for `"text/event-stream"`
- I am a bit confused here, since we are only streaming **in** data from them, not streaming anything **out**. So should it be`...bearer(at.token).contentType(MediaTypes.EventStream)` instead of `...bearer(at.token).header(Accept, EventStream)`??? I assume your tests are working, so I guess I'm missing something...

Reply:
Let me try with your approach, and replace code if it works.